File: reto-web/src/app/models/root.ts
import { Person } from "./person";

export interface Root {
    enum: string;
  id: string;
  person: Person;
  random: number;
  random_float: number;
  bool: boolean;
  date: string;
  regEx: string;
  elt: string;
  creationDate: string;
}

File: backend/DataTransformerService/src/main/java/com/intercorp/DataTransformerService/infrastructure/dataacces/Root.java
package com.intercorp.DataTransformerService.infrastructure.dataacces;


import com.fasterxml.jackson.annotation.JsonProperty;

import java.util.Map;

public record Root(
        String id,
        Person person,
        Integer random,
        Float random_float,
        Boolean bool,
        String date,
        String regEx,
        @JsonProperty("enum")
        String enumValue,
        String elt,
        Map<String, PersonDetails> personDetails
) {


}